command/connect/envoy/bootstrap_tpl.go

```diff
@@ -23,9 +23,9 @@ type BootstrapTplArgs struct {
 	// TLS.
 	AgentTLS bool
 
-	// AgentCAFile is the CA file to use to verify the local agent gRPC service if
+	// AgentCA is the CA to use to verify the local agent gRPC service if
 	// TLS is enabled.
-	AgentCAFile string
+	AgentCA []byte
```

Comment:
nit: if this is the raw certificate PEM I'd name it `AgentCAPEM` to be clear as it could be anything from the path to PEM to raw ASN1 etc...